Cargo.toml

```diff
@@ -1,62 +1,3 @@
-[package]
-name = "bitcoin"
-version = "0.29.0"
-authors = ["Andrew Poelstra <apoelstra@wpsoftware.net>"]
-license = "CC0-1.0"
-homepage = "https://github.com/rust-bitcoin/rust-bitcoin/"
-repository = "https://github.com/rust-bitcoin/rust-bitcoin/"
-documentation = "https://docs.rs/bitcoin/"
-description = "General purpose library for using and interoperating with Bitcoin and other cryptocurrencies."
-keywords = [ "crypto", "bitcoin" ]
-readme = "README.md"
-exclude = ["./test_data"]
-edition = "2018"
-
-# Please don't forget to add relevant features to docs.rs below
-[features]
-default = [ "std", "secp-recovery" ]
-rand = ["secp256k1/rand-std"]
-serde = ["actual-serde", "bitcoin_hashes/serde", "secp256k1/serde"]
-secp-lowmemory = ["secp256k1/lowmemory"]
-secp-recovery = ["secp256k1/recovery"]
-
-# At least one of std, no-std must be enabled.
-#
-# The no-std feature doesn't disable std - you need to turn off the std feature for that by disabling default.
-# Instead no-std enables additional features required for this crate to be usable without std.
-# As a result, both can be enabled without conflict.
-std = ["secp256k1/std", "bitcoin_hashes/std", "bech32/std"]
-no-std = ["hashbrown", "core2/alloc", "bitcoin_hashes/alloc", "secp256k1/alloc"]
-
-[package.metadata.docs.rs]
-features = [ "std", "secp-recovery", "base64", "rand", "serde", "bitcoinconsensus" ]
-rustdoc-args = ["--cfg", "docsrs"]
-
-[dependencies]
-bech32 = { version = "0.9.0", default-features = false }
-bitcoin_hashes = { version = "0.11.0", default-features = false }
-secp256k1 = { version = "0.24.0", default-features = false, features = ["bitcoin_hashes"] }
-core2 = { version = "0.3.0", optional = true, default-features = false }
-
-base64 = { version = "0.13.0", optional = true }
-bitcoinconsensus = { version = "0.20.2-0.5.0", optional = true }
-# Do NOT use this as a feature! Use the `serde` feature instead.
-actual-serde = { package = "serde", version = "1", default-features = false, features = [ "derive", "alloc" ], optional = true }
-hashbrown = { version = "0.8", optional = true }
-
-[dev-dependencies]
-serde_json = "<1.0.45"
-serde_test = "1"
-secp256k1 = { version = "0.24.0", features = [ "recovery", "rand-std" ] }
-bincode = "1.3.1"
-
-[[example]]
-name = "bip32"
-
-[[example]]
-name = "handshake"
-required-features = ["std"]
-
-[[example]]
-name = "ecdsa-psbt"
-required-features = ["std", "bitcoinconsensus"]
+[workspace]
+members = ["bitcoin"]
```

Comment:
Missing `bitcoin-internals`
Maybe the *directory* should be called just `internals` (crate still named `bitcoin-internals`)